custom/icds_reports/reports/awc_infrastracture.py

```diff
@@ -81,15 +81,15 @@ def get_data_for(month, filters):
                 }
             ],
             [
-                {
-                    'label': _('AWCs with Electricity'),
-                    'help_text': _('Percentage of AWCs with access to electricity'),
-                    'percent': 0,
-                    'value': 0,
-                    'all': 0,
-                    'format': 'percent_and_div',
-                    'frequency': 'month'
-                },
+                # {
```

Comment:
Why didn't you just remove this?

Reply:
@emord 
because at one of the meetings someone said that the indicators with gray background will be added in the future.